.github/ISSUE_TEMPLATE/Software-Project-Contribution.md

```diff
@@ -100,9 +101,19 @@ As a project onboarding into FINOS, you will need to familiarize yourself and yo
  - [FINOS Project/Standards Governance](https://community.finos.org/docs/governance)
  - [FINOS Project Lifecycle](https://community.finos.org/docs/governance/Software-Projects/project-lifecycle)
 
-## Record The Contribution (FINOS Infra)
-- [ ] (optional) Identify and Assign FINOS Strategic Advisor
-- [ ] Submit contribution to LF by opening a ticket via https://jira.linuxfoundation.org/browse/SS and marking contribution as "Exploratory"; attach a summary of the **Business Problem** and **Proposed Solution** (above) of the project.
```

Comment:
Moved to the internal Asana task. Note that contribution is submitted via PCC (not Jira).